docs/updating-diagrams.md

```diff
@@ -27,30 +27,14 @@ locally hosted server by changing the value of `plantuml.server`.
 
 When all changes have been made:
 
-6. With the diagram file open, using `Ctrl + shift + P` on Windows or `command + shift + P` on MacOS and searching "
-   export current diagram", choose the `PlantUML: Export Current Diagram` option.
+6. Push your code back to the repository, the [Generate PlantUML Diagrams GitHub Action](#automatic-diagram-generation) will generate the required `.png`
+   files automatically.
 
-This will display options for different file types to export to.
+## Automatic Diagram Generation
 
-7. Choose the `png` option.
+A GitHub Action named [Generate PlantUML Diagrams](https://github.com/ministryofjustice/hmpps-integration-api/blob/main/.github/workflows/generate-plantuml-diagrams.yml)
```

Comment:
```suggestion
We have a GitHub Actions workflow named [Generate PlantUML Diagrams](https://github.com/ministryofjustice/hmpps-integration-api/blob/main/.github/workflows/generate-plantuml-diagrams.yml)
```

From my understanding of what's described in https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions#workflows and https://docs.github.com/en/actions/creating-actions/about-custom-actions, as it's not published to the GitHub Marketplace and installable/usable at a step level in a workflow I don't think we can say it's a GitHub Action _yet_.

Reply:
Fair comment and some nice research done there Ting